Update web

makeclean opened the pull request:
This PR updates documentation for the Svalinn DAGMC website. It includes changes relating to
- Remove all mentions of CGM in build instructions
- Reintroduces "legacy" workflow documentation
- Introduces DAG tally Documentation
- Gives recommendations of producing DAGMC models using Tim Bohm's recommended workflow
- Add some documentation regarding watertightness

There may be some things to change, but all comments are welcome.

kkiesling commented on doc/usersguide/cubit_trelis_plugin.rst:
```diff
@@ -0,0 +1,37 @@
+
+OS/X Install Instructions
+-------------------------
+Obtain the `DAGMC Plug-in <https://drive.google.com/open?id=0B_tNonFwhOqQQkdhYWxWN1drVGM>`_
```
I don't think this link is public, so perhaps it should be moved to a spot like the linux plugin at go.wisc.edu

ljacobson64 commented on doc/usersguide/quick_start.rst:
```diff
@@ -1,25 +1,31 @@
+  2. Simplify and defeature your CAD as your see fit
+  3. Imprint and merge the model
+  4. Export your model to DAGMC
+  5. Use make_watertight, a tool that is build alongside DAGMC, to seal your geometry
```
this should say "built" not "build"

ljacobson64 commented on doc/usersguide/workflow/cubit_trelis_workflow.rst:
```diff
@@ -0,0 +1,302 @@
+-----------------------
+
+The general workflow for the production of models for analysis using DAGMC
+looks like that show in the Figure below.
```
should be "shown" not "show"

ljacobson64 commented on doc/usersguide/workflow/cubit_trelis_workflow.rst:
```diff
@@ -0,0 +1,302 @@
+         volofvolcubit.py-examine small volumes (vol<=0.0000 might need to autoheal)
+4. create pre-imprint/merge table of volume of volumes (use volofvolcubit.py)
+   Imprint model **imprint body all**
+5. Merge model
```
command is **merge all**

kkiesling commented on doc/usersguide/quick_start.rst:
```diff
@@ -1,25 +1,31 @@
-9. Run the problem as you see fit
+  1. Import your model into Trelis/Cubit, assigning volumes to groups of the same material, e.g. 
+     adding all volumes that are to be steel to one group and so on
+  2. Simplify and defeature your CAD as your see fit
```
"as _you_ see fit"

makeclean commented:
Ok I've just added some new commits and moved the gallery around so it appears on the main page, I think I'm pretty happy with this, i would appreciate if someone could build the docs and see what they think. All comments are welcome, I would like to get this live asap. How about @ljacobson64 could you take another look?

ljacobson64 commented:
So the slideshow looks great, but it makes it impossible to read anything below it while it's running because all the images have a different height.

ljacobson64 commented:
I think that either someone should update the Upcoming Features page or we should remove the link to it, because right now it's pretty bare-bones and I don't think it gives a very good idea of the upcoming features we're actually working on.

ljacobson64 commented:
I see a few references to something called "AutomatedCubitConversion" in multiple pages but I don't know what that is and I can't find any information on it anywhere. My instinct is that it probably isn't something we use anymore.

Pshriwise commented:
@ljacobson64 re:slideshow

Yeah that implementation of the slideshow wasn't really meant to be thrown into the middle of a bunch of text. I'll see what I can do about maintaining the image size and PR into this branch
